tracpro/polls/tests/test_tasks.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals
+
+import mock
+
+from tracpro.test import factories
+from tracpro.test.cases import TracProTest
+
+from .. import models
+from ..tasks import sync_questions_categories
+
+
+class TestPollTask(TracProTest):
+
+    @mock.patch.object(models.Question.objects, 'from_temba')
+    def test_sync_questions_categories(self, mock_question_from_temba):
```

Comment:
Can we also test that we create a new local Question if there's a new question in RapidPro?